Utils.py

```diff
@@ -616,6 +616,40 @@ def get_fuzzy_ratio(word1: str, word2: str) -> float:
         )
     )
 
+def get_intended_text(input_text: str, possible_answers) -> typing.Tuple[str, bool, str]:
```

Comment:
```suggestion
    )


def get_intended_text(input_text: str, possible_answers) -> typing.Tuple[str, bool, str]:
```
pep 8 specifies 2 blank lines between global function definitions